src/ngraph/op/fused/normalize_l2.cpp

```diff
@@ -92,12 +82,12 @@ NodeVector op::NormalizeL2::decompose_op() const
     auto builder_bias_mode =
         (m_eps_mode == EpsMode::MAX) ? builder::BiasMode::MAX : builder::BiasMode::ADD;
     Output<Node> norm = builder::l2_norm(data, reduction_axes, m_eps, builder_bias_mode);
-    norm = make_broadcast_node(norm, data.get_shape(), 0);
+    norm = numpy_style_broadcast(norm, data.get_shape());
```

Comment:
It is key change which should fix problems with incorrect shapes

Reply:
Let's try to use `auto_broadcast` here.

Reply:
Thanks, Using `Divide` with param `AutoBroadcastType::NUMPY` gives expected results